### PR TITLE
Updates to animations for new gallery

### PR DESCRIPTION
**Pull request by matthewbirtch**
#### Summary
This PR suggests a few updates to the animations to improve the smoothness and timing of some of the new gallery updates.

1. Speeds up the transition for opening an image from the channel list
2. Speeds up the transition for closing the gallery as well
3. Increases the sensitivity of swiping down to close the gallery
4. Speeds up and smooths out the transition of swiping between images (left/right). 
5. Speeds up the transition of the footer sliding down when tapping the share icon

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-31049

#### Checklist
- [x] Has UI changes

#### Device Information
This PR was tested on: iOS simulator
I couldn't get Android up and running easily

### CODE REVIEW

File: app/utils/images.js [hunk 1]
```diff
@@ -94,21 +94,22 @@ export function openGalleryAtIndex(index, files) {
...
+                duration: 150,
                 interpolation: {mode: 'decelerate'},
             };
 
             if (Platform.OS === 'ios') {
                 contentPop.translationY = {
                     from: 0,
                     to: windowHeight,
-                    duration: 300,
+                    duration: 150,
```
Comment (enahum) [inline app/utils/images.js:112]: @matthewbirtch no changes for android ?

**Comment by esethna** [comment]
@matthewbirtch are you a dev too now? ;) I can't seem to load the test build but if you're happy with it please don't block it on my review

File: app/utils/images.js (hunk 1 above)
Comment (matthewbirtch) [reply app/utils/images.js:112]: I made this update

**Comment by matthewbirtch** [comment]
@enahum I just tested on android and it appears that the transition from channel to gallery doesn't work? Although the transition does seem to work when closing the gallery. This appears to be an issue on the production app, so should we file a ticket for this?

**Comment by enahum** [comment]
@matthewbirtch file the report at wix/react-native-navigation ;)

Either that and wait for the fix or we would be forced to downgrade. Not sure if they already fixed it though

**Comment by enahum** [comment]
Maybe it broke cause of this https://github.com/wix/react-native-navigation/commit/d1355780bbbfe9f0cea0488dd143fc3089fb4107

**Comment by matthewbirtch** [comment]
Sorry, changing a few animation values is in my wheelhouse, but going much further beyond that, I'm getting out of my depth with the app.

**Comment by enahum** [comment]
@matthewbirtch I've been looking into this, and apparently there is a race condition in the navigation library when it comes to shareElementTransitions on Android, the code is being executed before the Element has been created, that is why only the default alpha transition is being applied.

I've tried many different things to make this thing work, but I do not really understand the entirety of the library code for this, so I guess that transition on Android will remain as is until it gets fixed.

**Comment by matthewbirtch** [comment]
> @matthewbirtch I've been looking into this, and apparently there is a race condition in the navigation library when it comes to shareElementTransitions on Android, the code is being executed before the Element has been created, that is why only the default alpha transition is being applied.
> 
> I've tried many different things to make this thing work, but I do not really understand the entirety of the library code for this, so I guess that transition on Android will remain as is until it gets fixed.

Ok, thanks for the follow-up. Do you think we should think about another library for shared element transitions? This one looks pretty decent (https://github.com/IjzerenHein/react-native-shared-element), but I don't know if it would be compatible with the navigation library.

**Comment by enahum** [comment]
@matthewbirtch that one is used without navigation, which does not work for us as this is another screen. There is this same one but for the other nav library which we looked at but is less perfomant than the one we are using.

Is just a matter to report it and wait for a fix I don't think this small issue on Android is enough for us to spend time either changing the way we do share element transitions or changing the whole navigatikn ;)

**Review by enahum — APPROVED** [review]

**Review by josephbaylon — APPROVED** [review]
LGTM. User experiences the transition and sensitivity changes.

Tested on both iPhone X (14.3) and Pixel 3 XL (11) devices.

**Comment by josephbaylon** [comment]
@matthewbirtch @enahum Is this still waiting on a separate PR change?

**Comment by matthewbirtch** [comment]
> @matthewbirtch @enahum Is this still waiting on a separate PR change?

No, not that I know of. I just don't have the power to merge :)

**Pull request merged**